debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/antlr/listener/AlterTableParserListener.java

```diff
@@ -161,7 +169,7 @@ public void exitAlterByChangeColumn(MySqlParser.AlterByChangeColumnContext ctx)
             Column column = columnDefinitionListener.getColumn();
             tableEditor.addColumn(column);
             String newColumnName = parser.parseName(ctx.newColumn);
-            if (newColumnName != null && !column.name().equals(newColumnName)) {
+            if (newColumnName != null && column.name().compareToIgnoreCase(newColumnName) != 0) {
```

Comment:
@jpechane this not directly related to DBZ-771, but I noticed and fixed along the way. This would make things fail when giving a column's name in different casing during ALTER TABLE then given within the original editor.

Reply:
@gunnarmorling What are adavatages here over `equalsIgnoreCase()`?

Reply:
None really, apart from the fact that I wasn't aware of `equalsIgnoreCase()` :) I'll change it.